src/EditorFeatures/VisualBasicTest/Structure/MultiLineIfBlockStructureTests.vb

```diff
@@ -9,37 +9,119 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.Outlining
         Inherits AbstractVisualBasicSyntaxNodeStructureProviderTests(Of MultiLineIfBlockSyntax)
 
         Friend Overrides Function CreateProvider() As AbstractSyntaxStructureProvider
-            Return New MultiLineIfBlockStructureProvider()
+            Return New MultiLineIfBlockStructureProvider(IncludeAdditionalInternalSpans:=True)
```

Comment:
i wouldn't do this.  That's not how our tests normally behave.   They don'tnormally configure behavior of features (except through actual Option values.  This means the  test is not validating the actual provider instance that is used in real code.